src/main/java/seedu/duke/Backend.java

```diff
@@ -81,13 +81,14 @@ public static SecretMaster initialisation() {
      * @param database Current ArrayList of Secret.
      * @return ArrayList of Secret
      */
-    public static ArrayList<Secret> readAndUpdate(String[] input, ArrayList<Secret> database) throws InvalidURLException {
+    public static ArrayList<Secret> readAndUpdate(String[] input, ArrayList<Secret> database)
+            throws InvalidURLException {
         //create different password based on constructor
         //studentID
         if (input[0].equals("studentID")) {
             Secret secret = new StudentID(input[1], input[2], input[3]);
             database.add(secret);
-        } else if (input[0].equals("NUSNetID")) {
```

Comment:
There are a lot of assumptions to this. What if they accidentally misstype, for ecample NUSnetID? Do these hlep?